src/Integration/Helpers/SonarLintOutput.cs

```diff
@@ -18,10 +18,13 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System.ComponentModel.Composition;
 using Microsoft.VisualStudio.Shell;
 
 namespace SonarLint.VisualStudio.Integration
 {
+    [Export(typeof(ISonarLintOutput))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
```

Comment:
Here we go....I was afraid of the time this would come xD

Reply:
Magic, eh :)